problems/arrayMethods.js

```diff
@@ -64,15 +98,19 @@ const numbersOnly = () => {};
  * @returns {boolean} Whether or not all elements are the same.
  */
 
-const isAllSame = () => {};
+const isAllSame = (items) => {
+  return items.every(el => el === items[0])
+};
 
 /**
  * Takes in an array and returns whether or not all elements in the array are odd.
  * @param {number[]} nums
  * @returns {boolean} All odd?
  */
 
-const isAllOdd = () => {};
+const isAllOdd = (nums) => {
+  return nums.every(el => el % 2 !== 0)
```

Comment:
another way to write this is just "el % 2" without the !== 0 because el % 2 returns either 0, a falsey, or 1, a truthy